src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -110,15 +111,18 @@ public AppealController(AppealService appealService) {
         @ApiResponse(responseCode = "500", description = "Internal server error")
     })
     @GetMapping(value = "/{company-id}/appeals", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<Appeal> getAppealByPenaltyReference(@PathVariable("company-id") final String companyId,
+    public ResponseEntity<List<Appeal>> getAppealByPenaltyReference(@PathVariable("company-id") final String companyId,
```

Comment:
Now we are returning a list, could you rename this to `getAppealsByPenaltyReference`? 
Also update the swagger documentation above to reflect this